### PR TITLE
Documentation grammar updates

### PR DESCRIPTION
**Pull request by GMTechBen**

### CODE REVIEW

File: misago/core/migrations/0002_basic_settings.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def create_basic_settings_group(apps, schema_editor):
                 {
                     'setting': 'forum_index_title',
                     'name': _("Index title"),
-                    'description': _("You may set custon title on forum index by typing it here."),
+                    'description': _("You may set a custom title on forum index by typing it here."),
```
Comment (rafalp) [inline misago/core/migrations/0002_basic_settings.py]: Hey,

Sadly, its not that easy to just fix a typo in settings migration. You'll need to create new migration that runs `migrate_settings_group` with data structure containing current settings group (including corrected spelling) and remove the `_()` from around the strings in old migration.

You'll also need to regenerate language files for english language for things to stay in sync for translators.

If you want, I may do this myself in coming days to show you (and others) how settings migrations should be fixed.

**Comment by GMTechBen** [comment]
Now that these were manually merged, should I close this?

**Comment by rafalp** [comment]
Yup, thank you for stepping up to help!

In future, please split docs/readme typo changes and language strings changes in separate PR's. Its far easier to merge in docs changes, than it is to merge changes that touch translation strings.

**Comment by GMTechBen** [comment]
Sounds good, happy to help!

**Pull request closed without merging**